src/pages/public_registration/action_accordions/DoiRequestAccordion.tsx

```diff
@@ -100,8 +108,10 @@ export const DoiRequestAccordion = ({
     }
   };
 
-  const hasMismatchingDoiRequest =
-    (isPendingDoiRequest && !registration.doi) || (isClosedDoiRequest && !!registration.doi);
+  const waitingForReservedDoi =
+    isPendingDoiRequest && !registration.doi && registration.status !== RegistrationStatus.Published;
```

Comment:
I dag vil man få en reservert DOI når man ber om DOI for en publisert Registrering. Dette tror jeg er feil (har meldt til JEG), hvis ikke bør ene endringen her fjernes
```suggestion
    isPendingDoiRequest && !registration.doi;
```

Reply:
stemmer at reservert DOI aldri skal vises når man ber om en DOI